hudi-sync/hudi-hive-sync/pom.xml

```diff
@@ -121,6 +121,17 @@
       <artifactId>hive-common</artifactId>
       <version>${hive.version}</version>
     </dependency>
+    <dependency>
```

Comment:
Can you remove this dependency as the change is in hudi-hadoop-mr package and hive-exec is already defined as dependency in hudi-hadoop-mr /pom.xml